config-connector/solutions/iam/kpt/service-account/Kptfile

```diff
@@ -0,0 +1,30 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: .
+packageMetadata:
+  shortDescription: sample description
+openAPI:
+  definitions:
+    io.k8s.cli.setters.service-account-name:
+      x-k8s-cli:
+        setter:
+          name: service-account-name
+          value: ${SERVICE_ACCOUNT?}
+          setBy: PLACEHOLDER
+      description: The name of the new IAM service account.
+    io.k8s.cli.setters.iam-member:
+      x-k8s-cli:
+        setter:
+          name: iam-member
+          value: ${IAM_MEMBER?}
```

Comment:
This could be something like 'user:name@example.com'.

Reply:
We can add 'user:name@example.com' in the description or readme, but we shouldn't make it the default value. It will confuses customers because they might think this is a valid value.

Reply:
Made the service account to package-default while left the iam-member value as a PLACEHODER.